src/coreclr/jit/emit.h

```diff
@@ -1869,14 +1869,20 @@ class emitter
         // beginning of the function -- of the target instruction of the jump, used to
         // determine if this jump needs to be patched.
         unsigned idjOffs :
-#if defined(TARGET_XARCH)
-            29;
-        // indicates that the jump was added at the end of a BBJ_ALWAYS basic block and is
+#if defined(TARGET_AMD64)
+            28;
+        // Indicates the jump was added at the end of a BBJ_ALWAYS basic block and is
```

Comment:
fwiw, all this bit fiddling is silly on 64-bit, where we're going to have 32 bits of padding after the `unsigned` fields, in which case we should just move all the bits to the end and make them `bool :1` fields. Maybe not for 32-bit, though.

But that shouldn't be done in this PR :)